### PR TITLE
feat(e2e): add new method to get active menu item

### PR DESCRIPTION
**Pull request by cedehe**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

We cannot get the current menu item.

**What is the new behavior?**

New method to get the active menu item in the side panel menu.

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://372.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://372.talend.surge.sh)

File: packages/components/e2e/component-objects/src/main/java/com/talend/component/SidePanel.java [hunk 1]
```diff
@@ -23,6 +23,8 @@
 
     static final String MENU_ITEMS_SELECTOR = ".tc-side-panel-list-item span";
 
+    static final String MENU_ITEM_ACTIVE_SELECTOR = ".tc-side-panel-list-item.active span";
```
Comment (frassinier) [inline packages/components/e2e/component-objects/src/main/java/com/talend/component/SidePanel.java]: DRY, define private constant for `.tc-side-panel-list-item`

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://372.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://372.talend.surge.sh)

**Pull request merged**